presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -587,6 +588,18 @@ public FeaturesConfig setSpillEnabled(boolean spillEnabled)
         return this;
     }
 
+    public boolean isSpillEncryptionEnabled()
+    {
+        return spillEncryptionEnabled;
+    }
+
+    @Config("experimental.spill-encryption-enabled")
```

Comment:
Why experimental? What's the plan to make it mainstream?

Reply:
I was trying to match the convention for other spill-to-disk related properties and the flag to enable / disable spilling altogether is also marked `experimental`.